source/patterns/@aws-solutions-constructs/aws-route53-apigateway/test/test.route53-apigateway.test.ts

```diff
@@ -292,45 +300,32 @@ test("Test for A record creation in Public Hosted Zone ", () => {
         "REGIONAL"
       ]
     },
-    Name: "LambdaRestApi"
+    Name: "RestApi"
   });
 });
 
 // --------------------------------------------------------------
 // Check for A record creation in Private Hosted Zone
 // --------------------------------------------------------------
-test("Test for A record creation in Private Hosted Zone ", () => {
+test("Integration test for A record creation in Private Hosted Zone ", () => {
   // Initial Setup
   const stack = new cdk.Stack();
   deployApi(stack, false);
 
   // Assertions
-  expect(stack).toHaveResource("AWS::CertificateManager::Certificate", {
-    DomainName: "www.test-example.com",
-    DomainValidationOptions: [
-      {
-        DomainName: "www.test-example.com",
-        HostedZoneId: {
-          Ref: "newzone1D011936",
-        },
-      },
-    ],
-    ValidationMethod: "DNS",
-  });
-
   expect(stack).toHaveResourceLike("AWS::Route53::RecordSet", {
```

Comment:
If this is supposed to be an Integration test, then we should strive to use toHaveResource and the full definition rather than toHave ResourceLike.